lib/trento.pm

```diff
@@ -271,6 +271,15 @@ sub config_cluster {
     $variables{SSH_KEY_PRIV} = SSH_KEY;
     $variables{SSH_KEY_PUB} = $ssh_key_pub;
     $variables{SCC_REGCODE_SLES4SAP} = $scc;
+
+    $variables{HANA_ACCOUNT} = get_required_var("TRENTO_QESAPDEPLOY_HANA_ACCOUNT");
+    $variables{HANA_CONTAINER} = get_required_var("TRENTO_QESAPDEPLOY_HANA_CONTAINER");
+    if (get_var("TRENTO_QESAPDEPLOY_HANA_TOKEN")) {
+        $variables{HANA_TOKEN} = get_required_var("TRENTO_QESAPDEPLOY_HANA_TOKEN");
```

Comment:
We should have one openQA TOKEN variable, not TRENTO_ HANASR_ TOKEN.

Reply:
yes, it is a wider topic. For the moment, the agreement is that the variables in the conf.yaml templates in https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/data/sles4sap/qe_sap_deployment all and only belong to the user (Trento, HanaSR, ...)

FYI @lpalovsky